lib/Message.php

```diff
@@ -81,7 +81,7 @@ function getBodyAsString() : string {
             return ob_get_clean();
         }
         $contentLength = $this->getHeader('Content-Length');
-        if (null === $contentLength) {
+        if (null === $contentLength || empty($contentLength)) {
```

Comment:
0 is not the same as not specifying the parameter. In this case we stream the whole body if no $contentLength was expected. Setting to to 0 would mess this up.

@sash04ek this has still an issue though, because `empty()` returns `true` for `0`, which is _definitely_ not what we want.